test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -697,6 +723,7 @@ public class HttpConnectionDispatcherTests
             var connection = manager.CreateConnection();
             var dispatcher = new HttpConnectionDispatcher(manager, new LoggerFactory());
             var context = new DefaultHttpContext();
+            context.Features.Set<IHttpResponseFeature>(new ResponseFeature());
```

Comment:
Because I started to use RegisterForDispose and the `DefaultHttpContext` throws. I plan to fix this in HttpAbstractions because it's a troll.